src/requestBuilder.js

```diff
@@ -2,6 +2,17 @@
 
 var merge = require('./functions/merge');
 
+function sortObject(obj) {
```

Comment:
I put it in this file instead of functions as I wasn't sure it has uses outside of here